tests/PHPUnit/Integration/ArchiveWithNoVisitsTest.php

```diff
@@ -78,6 +85,28 @@ public function test_getIdSitesToArchiveWhenNoVisits_CanBeUsedToTriggerArchiving
         $this->assertEquals($expectedMethodCalls, ArchiveWithNoVisitsTest_MockArchiver::$methodsCalled);
     }
 
+    public function test_PluginArchiver_CanBeUsedToTriggerArchiving_EvenIfSiteHasNoVisits()
+    {
+        PluginsArchiver::$archivers['VisitsSummary'] = 'Piwik\Tests\Integration\ArchiveWithNoVisitsTest_MockArchiver';
+
+        ArchiveWithNoVisitsTest_MockArchiver::$runWithoutVisits = true;
```

Comment:
Ok got it now, maybe for clarity you could split the other test in two? Since when we will deprecate the event we might remove the other test without seeing it was testing another useful case. LGTM otherwise :+1: